indexer-app/src/cmr/indexer/data/concepts/granule.clj

```diff
@@ -204,7 +204,7 @@
             :revision-date-doc-values revision-date
             :downloadable downloadable
             :browsable browsable
-            :created-at created-at
+            :created-at (or created-at revision-date)
```

Comment:
I forgot what we need to do when having to rebuild granule indexes... like split it into two sprints or sth?

Reply:
This only needs a reindexing of all granules.